Userland/Applications/Browser/BookmarksBarWidget.cpp

```diff
@@ -99,72 +99,84 @@ class BookmarkEditor final : public GUI::Dialog {
 
 static BookmarksBarWidget* s_the;
 
-BookmarksBarWidget& BookmarksBarWidget::the()
-{
-    return *s_the;
-}
-
-BookmarksBarWidget::BookmarksBarWidget(DeprecatedString const& bookmarks_file, bool enabled)
+ErrorOr<NonnullRefPtr<BookmarksBarWidget>> BookmarksBarWidget::try_create(DeprecatedString const& bookmarks_file, bool enabled)
```

Comment:
thereʼs a tendency to avoid `try_` prefix, because of `TRY` macro usage. `create` should be enough in that case.

(same elsewhere)

Reply:
@AtkinsSJ wrote in #17130 to adopt the `try_create()` because

>as that is what is expected by other code that constructs widgets

And as far as I can read in `Userland/Libraries/LibGUI/Window.h` the `set_main_widget` function looks for the `try_create()` on the T template argument. 

For example `Userland/Applications/Browser/Tab.cpp:758` the `ConsoleWidget` is instantiated with the above `set_main_widget()`:

```
console_window->set_main_widget<ConsoleWidget>()
```

What do you think?

Reply:
well… stick to what maintainers are willing to accept! 😁